lib/js/patterns/modal.js

```diff
@@ -42,8 +42,8 @@ function modal () {
 
   function openModal (modalId) {
     bus.emit('modal:close');
-    if (!modalId) return;
-    var modal = document.querySelector(`.js-modal[data-modal="${modalId}"]`);
+    if (!modalId.id) return;
```

Comment:
`if (!modalId || !modalId.id)` would probably be safer here. If they don't pass anything, then this will throw an error because you're accessing a property on a null object.